source/security/index.html.md.erb

```diff
@@ -7,20 +7,30 @@ weight: 140
 
 ## Reporting vulnerabilities
 
-If you believe GOV.UK Pay security has been breached, contact us immediately at [govuk-pay-support@digital.cabinet-office.gov.uk] (mailto:govuk-pay-support@digital.cabinet-office.gov.uk). If you are a live user and the suspected breach is severe, consider using the urgent contact details provided to your service manager.
+If you believe GOV.UK Pay security has been breached, contact us immediately
+at [govuk-pay-support@digital.cabinet-office.gov.uk]
+(mailto:govuk-pay-support@digital.cabinet-office.gov.uk). If you are a live
+user and the suspected breach is severe, consider using the urgent contact
+details provided to your service manager.
```

Comment:
"... when you signed up to GOV.UK Pay"?

Will this always be applicable?

Reply:
Yes, we'll still continue to privately give out the P1 emergency email (which triggers a PD alert), although in theory it might not be to a service manager (if there's not a service manager)..